js/app.js

```diff
@@ -1 +1,118 @@
-"use strict";
\ No newline at end of file
+"use strict";
+
+// ---------Task 1 
+
+// const colors = ['red', 'green', 'blue'];
+// console.log(colors.length);
+
+
+// ---------Task 2
+
+// const animals = ['monkey', 'dog', 'cat'];
+// console.log(animals[animals.length - 1]);
```

Comment:
second way arr.pop()

Reply:
I now, but .pop() removes the element from the array. If it's not a problem, we can use .pop()